cumulusci.yml

```diff
@@ -876,6 +876,26 @@ flows:
             4:
                 task: run_tests
 
+    config_trial_complete:
+        steps:
+            # TODO: Update once we can test with a Beta instead of 2GP
+            #1:
+            #    flow: dependencies
+            2:
+                flow: install_2gp_commit #task: install_managed
+            3:
+                task: deploy_post
+                options:
+                    unmanaged: False
+            4:
+                flow: config_trial
+            5:
+                flow: enable_rd2_managed
```

Comment:
@lparrott I think we can just use `enable_rd2` here. I ran it against a managed org and it went through fine. It doesn't appear that the `managed` property is needed to warrant a separate flow for this. If we could just use `enable_rd2` that would help avoid confusion around why that flow has a `managed` suffix and the others don't.